test/nodes/test_retriever.py

```diff
@@ -38,6 +38,7 @@
 
 
 # TODO check if we this works with only "memory" arg
+@pytest.mark.integration
```

Comment:
In theory these markers make sense, but due to some recent changes in how we run tests in CI it's best for now not to add these markers to the existing tests. Can you revert this change? Sorry for that!